parquet-thrift/src/main/java/parquet/hadoop/thrift/ThriftReadSupport.java

```diff
@@ -73,6 +73,12 @@ public static void setRecordConverterClass(JobConf conf,
     conf.set(RECORD_CONVERTER_CLASS_KEY, klass.getName());
   }
 
+  public static void setRecordConverterClass(Configuration conf,
```

Comment:
remove the duplication of setRecordConverterClass(JobConf conf) by calling this method there